src/platform/workbench/test/vscode-node/workbenchServiceImpl.test.ts

```diff
@@ -7,15 +7,15 @@ import * as assert from 'assert';
 import { WorkbenchServiceImpl } from '../../vscode/workbenchServiceImpt';
 
 suite('HostServiceImpl', () => {
-	test.skip('getAllCommands', async () => { // TODO@TylerLeonhardt
+	test('getAllCommands', async () => {
 		const envService = new WorkbenchServiceImpl();
 		const commands = await envService.getAllCommands();
 		assert.ok(Array.isArray(commands));
 		assert.ok(commands.length > 0);
 		assert.ok(commands[0].label);
 		assert.ok(commands[0].command);
 		assert.ok(commands[0].keybinding);
-	});
+	}).timeout(10000); // Give this test a little bit more time due to potential delays in fetching commands
```

Comment:
That's a very long timeout, we don't want any delays near that in unit tests which should run in milliseconds?